src/qt/walletframe.cpp

```diff
@@ -3,20 +3,19 @@
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 #include <qt/walletframe.h>
-#include <qt/walletmodel.h>
 
-#include <qt/bitcoingui.h>
+#include <qt/walletmodel.h>
```

Comment:
1f913ac9a00b982c997e70603ba62eebbe4c9d03

Looks like this is the only change needed?